guidelines/enhancement_template.md

```diff
@@ -115,6 +117,16 @@ bogged down.
 
 Include a story on how this proposal will be operationalized:  lifecycled, monitored and remediated at scale.
 
+### API Extensions
```

Comment:
Q: why is this section so far from the Impact of API Extensions section? Is there a reason to seperate them? Does doing so impact readability?

Reply:
the impact is in the details part, the listing is here, for easier reviewing.